Gems/Atom/Tools/AtomToolsFramework/Code/Source/GraphView/GraphView.cpp

```diff
@@ -47,11 +47,11 @@ namespace AtomToolsFramework
     GraphView::GraphView(
         const AZ::Crc32& toolId,
         const GraphCanvas::GraphId& activeGraphId,
-        const GraphViewConfig& graphViewConfig,
+        AZStd::shared_ptr<GraphViewSettings> graphViewSettingsPtr,
         QWidget* parent)
         : QWidget(parent)
         , m_toolId(toolId)
-        , m_graphViewConfig(graphViewConfig)
+        , m_graphViewSettingsPtr(graphViewSettingsPtr)
```

Comment:
Since you've already made a copy of the shared pointer, it's safe to `move` it here (a modest efficiency gain).